integration-tests/profiler/profiler.spec.js

```diff
@@ -14,7 +14,7 @@ const fs = require('fs/promises')
 const fsync = require('fs')
 const net = require('net')
 const zlib = require('zlib')
-const { Profile } = require('pprof-format')
+const { Profile } = require('../../vendor/dist/pprof-format')
```

Comment:
This is really annoying. I'm almost considering a build step just to not have to do this

Reply:
> I'm almost considering a build step just to not have to do this

That should be how we do this anyway, but unfortunately it is not currently possible as the code is not modularized, so we can't bundle individual features properly, and bundling a single output is unacceptable because of Serverless.